src/software/ai/hl/stp/tactic/kick/kick_fsm_test.cpp

```diff
@@ -13,27 +13,32 @@ TEST(KickFSMTest, test_transitions)
                                           .kick_speed_meters_per_second = 1.2};
 
     HFSM<KickFSM> fsm;
-    // Start in GetBehindBallFSM state's idle_state
+
+    // Start in GetBehindBallFSM state's get_behind_ball_state
     EXPECT_TRUE(fsm.is(boost::sml::state<GetBehindBallFSM>));
     EXPECT_TRUE(fsm.is<decltype(boost::sml::state<GetBehindBallFSM>)>(
```

Comment:
What's the difference between the two EXPECT_TRUE statements?

Reply:
the first line checks that the fsm is in _any_ substate of GetBehindBallFSM and the second line checks that it's in a particular substate of the FSM.